docker-compose-canary.yml

```diff
@@ -11,6 +11,8 @@ services:
   extends:
    file: ./modules/docker-compose.yml
    service: guacamole-server
+  container_name: "guacamole-server"
+  image: nanocloud/guacamole-server:0.4.0
```

Comment:
This is the canary docker-compose file.
Image should be nanocloud/guacamole-server without any tags (we want to download the latest)